src/lava/magma/runtime/runtime.py

```diff
@@ -64,6 +64,13 @@ def __init__(self,
         self.service_to_runtime_data: ty.Iterable[CspRecvPort] = []
         self.runtime_to_service_data: ty.Iterable[CspSendPort] = []
 
+    def __del__(self):
+        """On destructor call automatically
```

Comment:
Alternative docstring suggestion: "On destruction, terminate Runtime automatically to free compute resources."